gateway/app/endpoints/quality_on_demand/delete_device_session.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import APIRouter
+
+from app.drivers.quality_on_demand import QodInterfaceDep
+
+router = APIRouter()
+
+
+@router.delete("/sessions/{sessionId}", response_model_exclude_none=True)
```

Comment:
Remove the `response_model_exclude_none` since its not doing anything and set the status code to 204 No Content